airbyte-webapp/src/components/JobItem/JobItem.tsx

```diff
@@ -63,24 +63,25 @@ export const JobItem: React.FC<JobItemProps> = ({ shortInfo, job }) => {
   };
 
   useEffectOnce(() => {
-    if (linkedJobId) {
-      scrollAnchor.current?.scrollIntoView({
-        behavior: "smooth",
-        block: "start",
-      });
+    if (linkedJobId === String(getJobId(job))) {
+      // We need to wait here a bit, so the page has a chance to finish rendering, before starting to scroll
+      // since otherwise this scroll won't really do anything.
+      const timeout = window.setTimeout(() => {
+        scrollAnchor.current?.scrollIntoView({
+          behavior: "smooth",
```

Comment:
Nit: Not a huge fan of smooth behavior because it feels like my page is being scrolled by a ghost!

Reply:
Happy to remove it. Thought especially with the delay it might not suddenly jump, but I don't have any strong feelings in either direction :)